docs/release-notes/v0.6.0.rst

```diff
@@ -99,7 +99,8 @@ CLI API
      title = "{pyhf: v0.6.0}",
      version = {0.6.0},
      doi = {10.5281/zenodo.1169739},
-     url = {https://github.com/scikit-hep/pyhf},
+     url = {https://doi.org/10.5281/zenodo.1169739},
+     note = {https://github.com/scikit-hep/pyhf/releases/tag/v0.6.0}
```

Comment:
This is some revisionist history as this is changing release notes, but as this will only affect future views of the release notes I don't think this is too terrible.